lessonbuilder/api/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDao.java

```diff
@@ -196,6 +196,8 @@
 
 	public SimplePage getPage(long pageId);
 
+	public String getPageUrl(long pageId);
```

Comment:
There's a fair bit of code in SimplePageToolDaoImpl that isn't directly db related, so it seemed a fair place to put it - in the spirit of the original author, so to speak. I can create a service called LessonBuilderUrlService if that would be better.